test/unit/src/math/ColorManagement.tests.js

```diff
@@ -2,15 +2,33 @@
 
 import { ColorManagement } from '../../../../src/math/ColorManagement.js';
 
+import { CONSOLE_LEVEL } from '../../utils/console-wrapper.js';
+
 export default QUnit.module( 'Maths', () => {
 
 	QUnit.module( 'ColorManagement', () => {
 
 		// PROPERTIES
+		QUnit.test( 'enabled', ( assert ) => {
+
+			assert.ok(
+				ColorManagement.enabled == false,
```

Comment:
Shouldn't we use `===` instead of `==`?

Reply:
💪🏻 yes, should do that across all the tests in the next pass.